CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing
+
+## Content publishing checklist
+When you are ready for your pull request to be reviewed, make sure to go through this list so the reviewer can focus on giving feedback on the content and not have to nitpick.
+
+- [ ] Captalised B for protocol
+- [ ] Lowercase b for currency
+- [ ] Hyphenated and lowercase image file names
+- [ ] Social image in frontmatter is 1200px x 630px
+- [ ] Heavily textured images/photos are `.jpg`
+
```

Comment:
Some minor things I always tend to change that I am unsure is appropriate for this lists:

- Instead of 'funds' use 'bitcoin.'
- Instead of seed phrase, seed, mnemonic etc. use recovery phrase.

Reply:
Just sounds more pro bitcoin and appropriate tbh

Reply:
I don't think that's something that happens often and is relevant as much tbh, let's try to have a list of things that happen quite a lot and that will help us with the review process.

Reply:
I've changed a funds to bitcoin in almost every review I've done. Changed many-a recovery phrase also

Reply:
What words to use when could be an ever-growing list. Instead of calling them all out here, how about talking about that all in one place, like over here: https://bitcoin.design/guide/contribute/content-guidelines/#use-simple-language

We could then have one checklist item here that is 'Common terms are used correctly'